.github/workflows/pipeline.yml

```diff
@@ -1,20 +1,40 @@
 ---
 name: Pipeline
+
 on: [push]
+
 jobs:
   tests:
-    name: Unit Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
❓ is there a reason to change this to a specific version rather than sticking with the latest? This seems to add one more thing our team needs to keep on top of when 22.04 is removed from service down the line.

Reply:
Good point @jebentier 

Addressed the feedback in a follow-up PR at https://github.com/Invoca/invoca-utils/pull/26/files

And to the reason on why I decided to lock to a specific version in the first place is that the [tool](https://github.com/nektos/act) that I was using to run the GitHub actions is failing on `ubuntu-latest` but works fine with `ubuntu-22.04`. Moreover, it was never a good idea to lock to the latest which is unpredictable (as it could change underneath). Analogues to the docker latest tag. But to your feedback and thinking twice that's probably not applicable in this situation.